evap/staff/forms.py

```diff
@@ -291,7 +298,7 @@ def __init__(self, *args, **kwargs):
             disable_all_fields(self)
 
         if self.instance.pk:
-            responsible = self.instance.responsible_contributors[0]
+            responsible = self.instance.course.responsibles.first()
             self.fields['responsible'].queryset |= UserProfile.objects.filter(pk=responsible.pk)
             self.fields['responsible'].initial = responsible
```

Comment:
Can a single result have multiple responsibles?

Reply:
yes, every course can have multiple responsibles, we make no restrictions for single results.

Reply:
ah, no i see what you mean. this form here saves only one responsible person, because it is behaving like the form we had before. it drives me crazy because it will be changed in the next commit again - probably going for the 1:1 relation first and then making this 1:n wasn't the best way of doing this.
so once courses will be edited independent from evaluations (which will happen in the mentioned next commit) the course editing is separated from evaluations or single results and thus multiple responsibles can be set. not sure whether changing that here makes any sense at all.

Reply:
So, did I understand you correctly:
- We already had multiple responsibles for one course
- The single result form did not respect that and this change just keeps this as it was before.
?
If that's the case, please resolve, otherwise, I'm a bit confused :D

Reply:
yes, multiple responsibles per course were possible, but not in a single result.